types/jsrsasign/modules/KJUR/asn1/csr/CSRUtil.d.ts

```diff
@@ -11,7 +11,7 @@ declare namespace jsrsasign.KJUR.asn1.csr {
     }
 
     interface ParamResponse {
-        subject: { array?: IdentityResponse };
+        subject: { array?: Array<[{ type: string; value: string; ds: string }]>, str?: string };
```

Comment:
A single-value array in an array - is that correct?

Reply:
unfortunately yes, that way it is written, see https://github.com/kjur/jsrsasign/blob/1536f252d70d25db7ade80b31f57be26e52b26f9/src/asn1csr-1.0.js#L412

Reply:
Alright, thanks!